src/um/dwrite_1.rs

```diff
@@ -0,0 +1,719 @@
+// Copyright © 2017 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+//! Mappings for the content of dwrite_1.h
+
+use shared::basetsd::{ INT16, INT32, UINT16, UINT32, UINT8 };
+use shared::minwindef::{ BOOL, FLOAT };
+use um::dcommon::{ DWRITE_MEASURING_MODE };
+use um::dwrite::{ DWRITE_GLYPH_OFFSET, DWRITE_MATRIX, DWRITE_PIXEL_GEOMETRY,
+    DWRITE_RENDERING_MODE, DWRITE_SCRIPT_ANALYSIS,
+    DWRITE_SHAPING_GLYPH_PROPERTIES, DWRITE_TEXT_RANGE,
+    IDWriteBitmapRenderTarget, IDWriteBitmapRenderTargetVtbl, IDWriteFactory,
+    IDWriteFactoryVtbl, IDWriteFont, IDWriteFontCollection, IDWriteFontFace,
+    IDWriteFontFaceVtbl, IDWriteFontVtbl, IDWriteRenderingParams,
+    IDWriteRenderingParamsVtbl, IDWriteTextAnalysisSink,
+    IDWriteTextAnalysisSinkVtbl, IDWriteTextAnalysisSource,
+    IDWriteTextAnalysisSourceVtbl, IDWriteTextAnalyzer, IDWriteTextAnalyzerVtbl,
+    IDWriteTextLayout, IDWriteTextLayoutVtbl };
```

Comment:
Multiline imports should be block indented and fill up the column limit as much as possible.